views/assets/js/char-count.js

```diff
@@ -37,14 +37,9 @@
 	CharCount.prototype.getStyle = function(element, attributeName) {
 		var attributeValue = ''
 		if (document.defaultView && document.defaultView.getComputedStyle) {
-			attributeValue = document.defaultView
-				.getComputedStyle(element, '')
-				.getPropertyValue(attributeName)
+			attributeValue = document.defaultView.getComputedStyle(element, '').getPropertyValue(attributeName)
```

Comment:
Is this something we have developed? I am surprised gov.uk doesn't have this already developed